include/uk/essentials.h

```diff
@@ -104,6 +104,11 @@ extern "C" {
 #define __alias(old, new) \
 	extern __typeof(old) new __attribute__((alias(#old)))
 #endif
+
```

Comment:
Missing `.` at the end of the commit message

Reply:
fixed